barman/cli.py

```diff
@@ -67,8 +186,11 @@ def check_target_action(value):
     raise ArgumentTypeError("'%s' is not a valid recovery target action" % value)
 
 
-@named("list-servers")
-@arg("--minimal", help="machine readable output")
+@command([
+        argument("--minimal",
+                 help="machine readable output")
+        ],
```

Comment:
I think we need an `action="store_true"` here (there wasn't one in the original code, I suppose argh must have been adding that automatically).